test/e2e_node/restart_test.go

```diff
@@ -150,6 +150,10 @@ var _ = SIGDescribe("Restart [Serial] [Slow] [Disruptive]", func() {
 			if numCpus < 1 {
 				e2eskipper.Skipf("insufficient CPU available for kubelet restart test")
 			}
+			if numCpus > 18 {
+				// 950m * 19 = 1805 CPUs -> not enough to block the scheduling of another 950m pod
+				e2eskipper.Skipf("test will return false positives on a machine with >18 cores")
```

Comment:
I don't have a machine large enough to test if this actually works or not (since many pods are scheduled), but I figured I should include some sort of cutoff for possible false positives since I stopped scheduling 1 CPU per pod.

Reply:
/cc @aojea